x-pack/legacy/plugins/lens/public/xy_visualization_plugin/xy_config_panel.tsx

```diff
@@ -325,7 +278,10 @@ export function XYConfigPanel(props: VisualizationProps<State>) {
           onClick={() => {
             setState({
               ...state,
-              layers: [...state.layers, newLayerState(frame.addNewLayer())],
+              layers: [
+                ...state.layers,
+                newLayerState(state.preferredSeriesType, frame.addNewLayer()),
```

Comment:
The logic here feels wrong: I would expect that new layers use the existing series type unless the chart is already mixed.

Reply:
Right, I changed this to use the series type of the existing layers if it's the same for all layers.